index.html

```diff
@@ -136,6 +172,76 @@ <h3 class="card-head__heading">Mr. Eat</h3>
           >
         </div>
       </div>
+      <div class="card">
+        <div class="card__card-head">
+          <img
+            class="card-head__img"
+            src="https://images.unsplash.com/photo-1551782450-a2132b4ba21d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80"
+            alt=""
+          />
+          <h3 class="card-head__heading">Freddy Schilling</h3>
+        </div>
+        <div class="card__card-body">
+          <div class="card-body__left">
+            <p class="card-body__text">
+              Großartige, kreative und vor allem hochwertig handgemachte Burger
+              mit leckerer Pommes.Die Soßenauswahl ist Mega und alles wird
+              frisch gemacht. Definitiv ein Besuch wert!
+            </p>
+          </div>
+          <div class="card-body__right">
+            <span>⭐️⭐️⭐⭐ <small>4/4</small> </span>
+            <br />
+            <span> 💵 <small>Mittel bis sehr teuer</small></span>
+            <br />
+
+            <span>🥦🌱</span>
+          </div>
+        </div>
+        <div class="card__card-footer">
+          <a class="card__a" href="#"
+            ><button class="card__btn-route">Route</button></a
+          >
+          <a class="card__a" href="#"
+            ><button class="card__btn-info">Speisekarte</button></a
+          >
+        </div>
+      </div>
+      <div class="card">
+        <div class="card__card-head">
+          <img
+            class="card-head__img"
+            src="https://images.unsplash.com/photo-1577594990850-e843a8e91512?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2850&q=80"
+            alt=""
+          />
+          <h3 class="card-head__heading">Supersalad</h3>
+        </div>
+        <div class="card__card-body">
+          <div class="card-body__left">
+            <p class="card-body__text">
+              Geniale Salate, mit denen man Mittags wirklich satt werden
+              kann.Frische Zutaten, große Portionen und nur 5 min Fußgängig
```

Comment:
- "Mittags" must be lowercase

-the space  after "kann." is missing

- you wrote "5 min Fußgängig entfernt" but I wonder from where? also "Fußgängig" must be lowercase :)

Reply:
here is meant 5 min from Space, where we are.)I would also write it down. Or I just make it away